docs/architecture-decision-record/ADR-0008 - Choice of test framework for intradomain integration points when testing prior to dev.MD

```diff
@@ -0,0 +1,87 @@
+* We want a solution that does not require a full domain setup in an environment
+* The solution must be easy to set up and tear down as part of PR gate or deployment gates
+* We want a solution that are not affected by infrastructure, so test results are not biased by infrastructure issues
+
+Following use cases were identified, that needs to be covered by this test framework:
+
+* ### Triggering job in Ccomponents
+
+Being able to trigger, read status and verify output from the component.
+
+* ### Integration to Event Hub
+
+Verifying event is generated in event hub and that it contains expected data.
+
+* ### Integration to a ServiceBus
+
+Publishing a ChargeCommandReceived event from an azure function (MessageReceiver). Receiving this event from yet another az function (ChargeCommandReceiver).
+The test will be initiated by a HTTP POST call with JSON content.
+(No seed data required)
+
+* ### Databricks integration to a DeltaLake
+
+Write access - storing a time series in the DeltaLake (from Databricks)
+It is assumed that the test will be initiated by placing valid test data on an EventHub from which Databricks will fetch the time series.
+(No seed data required)
+
+* ### Integration to an Sql database
+
+Read/Write access for reading and storing a charge from an azure functions.
+The database must include seed data (MarketParticipants table), Charge test data is not required.
+
+## Considered Options
+
+* Option 1: Running integration tests in a Test Container
+* Option 2: Running integration tests in a PR environment
+* Option 3: Using a test framework like Squadron, that enables to test in a container setup with the ability to connect to services in Azure.
+
+* … <!-- numbers of options can vary -->
+
+## Decision Outcome
+
+For use cases regarding components like eventhub, service bus and Sql database we have implemented a Sqaudron based solution, where it is possible both local-hosted and in CI pipeline to ensure application-to-service  connections.
+
+***Regarding the use case for DeltaLake - it has not yet been finalized on which solution is most valuable, and therefore the decision to cover this is still pending.***
+
+### Positive Consequences <!-- optional -->
+
+* Besides testing component connections locally Squadron allows verification that components are connecting correctly to storage instances by creating a containerized setup where both application layer and service is connected in an infrastructure setup (excluding Terraform - further outline on this under negative consequences).
+
+* The solution is implemented to run both locally and in CI setup.
+
+* This adds to  even more risk elements to be covered prior to deployment, and mitigiates risks that essentially could cause instability and issues on environments and further out in the CD pipeline, which would enhance the complete lead time for functionality delivery.
```

Comment:
This sounds like a general positive consequence of implementing tests where one verify connections and data exchanges with external dependencies - and not a positive consequence specifically with the Squadron/containernized approach

Reply:
Will be moved to a more general topic on why we do this